Docs/staker section (#89)
stakewise/stakewise-docs: staker/stake-with-one-click.mdx

---

> title: Stake with One Click

Reviewer: Lets not use the term "One Click" especially as a product name.
Maybe Simple Staking & Vault Staking

description: Stake ETH and receive osETH automatically with one-click staking.
---

import Image from '@theme/IdealImage'
import Tooltip from '@site/src/components/Tooltip/Tooltip';

# Stake with One Click

One-click staking routes your assets through a [Meta Vault](/docs/vaults/meta-vaults) that distributes them across the best-performing sub-Vaults — currently just the [Genesis Vault](https://app.stakewise.io/vault/mainnet/0xac0f906e433d58fa868f936e8a43230473652885), with more to come.

> When staking, you'll automatically receive [osETH](/docs/ostoken/intro), which represents your staked assets plus accumulated rewards. osETH can be used across DeFi to boost your yield — check out the [StakeWise blog guides](https://blog.stakewise.io/guide) for examples, including strategies on Morpho, Fluid, Balancer, Gearbox, and more.

Reviewer: check out the [StakeWise blog guides](https://blog.stakewise.io/guide) for examples - I think its better to share the ecosystem page here.

> <Image img={require('./img/one_click_staking.png')} alt="StakeWise one-click staking explanation diagram" />

Reviewer: Remove the cursos logo on this image

:::

## How to Unstake?

Unstaking is just as simple — convert your osETH back to ETH directly in the StakeWise app whenever you want. If you have an active Boost position, switch to the **Unboost** tab to exit it before unstaking.

1. Open [app.stakewise.io](https://app.stakewise.io) and connect your wallet.
2. Click the **swap arrows** icon (top-left) to switch from staking to unstaking mode.
3. Select the **Unstake** tab and enter the amount of osETH you want to convert back to ETH.
4. Review the details.
5. Click **Unstake** and confirm the transaction in your wallet.
6. Once confirmed, your ETH (original stake plus accumulated rewards) will be returned to your wallet.

:::custom-info[Unstaking Times]
- If you unstake an amount equal to or less than what you staked, your request enters the exit queue — the wait time is shown in the **Exit duration** field.

> - If you unstake more than you staked (e.g. you also hold osETH in your wallet), the staked portion enters the exit queue while the rest is swapped instantly via <Tooltip content="CoW Swap is a DEX aggregation protocol that finds the best price across decentralized exchanges.">CoW</Tooltip>.

Reviewer: ```suggestion
- If you unstake more than you staked (e.g. you also bought osETH on the market), the staked portion enters the exit queue while the rest is swapped instantly via <Tooltip content="CoW Swap is a DEX aggregation protocol that finds the best price across decentralized exchanges.">CoW</Tooltip>.
```

> - If you never staked and only hold osETH, the swap happens instantly via CoW.

Reviewer: ```suggestion
- If you never staked and only bought osETH on the market, the swap happens instantly via CoW.
```

> - If you never staked and only hold osETH, the swap happens instantly via CoW.

Reviewer: Lets replace CoW here with an exchange.
